polkadot/xcm/xcm-executor/src/lib.rs

```diff
@@ -574,41 +573,45 @@ impl<Config: config::Config> XcmExecutor<Config> {
 		Ok(())
 	}
 
-	/// Calculates the amount of `self.asset_used_for_fees` required to swap for
-	/// `asset_needed_for_fees`.
+	/// Calculates the amount of asset used in `PayFees` or `BuyExecution` that would be
+	/// charged for swapping to `asset_needed_for_fees`.
 	///
 	/// The calculation is done by `Config::AssetExchanger`.
-	/// If `self.asset_used_for_fees` is not set, it will just return `asset_needed_for_fees`.
+	/// If `self.asset_used_in_buy_execution` is not set, it will just return
+	/// `asset_needed_for_fees`.
```

Comment:
```suggestion
	/// If neither `PayFees` or `BuyExecution` were not used, or no swap is required,
	/// it will just return `asset_needed_for_fees`.
```